runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -4653,3 +4657,22 @@ TR::CompilationInfoPerThreadRemote::clearPerCompilationCaches()
    clearPerCompilationCache(_resolvedMirrorMethodsPersistIPInfo);
    clearPerCompilationCache(_classOfStaticMap);
    }
+
+void
+TR::CompilationInfoPerThreadRemote::terminateClient(uint64_t clientId, TR::CompilationInfo* compInfo, J9VMThread* compThread)
```

Comment:
I feel that 'deleteClientSessionData' is more in line with the function actually does.